localchain_contract_addresses.env

```diff
@@ -4,3 +4,8 @@ CONTRACT_ADDRESS_PLASMA_FRAMEWORK=0xc673e4ffcb8464faff908a6804fe0e635af0ea2f
 CONTRACT_ADDRESS_ETH_VAULT=0x0433420dee34412b5bf1e29fbf988ad037cc5db7
 CONTRACT_ADDRESS_ERC20_VAULT=0x04badc20426bc146453c5b879417b25029fa6c73
 CONTRACT_ADDRESS_PAYMENT_EXIT_GAME=0x92ce4d7773c57d96210c46a07b89acf725057f21
+CONTRACT_SHA=835d10cbe8d7ef56feb745f71de638f85f572f7d
```

Comment:
Have you considered moving these into ENVs in Circle? I know it's slightly less deterministic because once an ENV is added you cannot view it, but it would enable changes to be made without a PR/rebasing etc.

Reply:
I think having these info here helps pinning/tracing elixir-omg code base to the contracts it's intended to support though. Very useful for local developments. 🎉

Reply:
yeah I think @unnawut is right, it would also mean that not anyone could bump contracts but only those that have admin rights on circleci or they'd have to ask for someone else to do it :|

Reply:
I just saw @jbunce's PR on another repo which seems similar. Looks like we already have it in elixir-omg: https://github.com/omisego/elixir-omg/blob/master/.circleci/ci_deploy.sh#L45